app/components/queries/sort_by_field_component.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= render(Primer::OpenProject::FlexLayout.new) do |flex| %>
+<%= render(Primer::OpenProject::FlexLayout.new(data: { 'test-selector': 'sort-by-field' })) do |flex| %>
```

Comment:
```suggestion
<%= render(Primer::OpenProject::FlexLayout.new(test_selector: 'sort-by-field' })) do |flex| %>
```

Nitpick, this can be just a system argument which will do that for you, and also hide it in production:

> Adds data-test-selector='given value' in non-Production environments for testing purposes.